### PR TITLE
docs(aws-kms): Use back tickets in documentation

### PR DESCRIPTION
**Pull request by michaelbrewer**
Currently using single quotes truncates the documentation. See screenshot below.

Screen Shot 2021-02-19 at 7 15 31 AM

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-kms/lib/key.ts [hunk 1]
```diff
@@ -322,13 +322,13 @@ export interface KeyProps {
...
+   * If the `@aws-cdk/aws-kms:defaultKeyPolicies` feature flag is set (the default for new projects),
    * this flag will always be treated as 'true' and does not need to be explicitly set.
    *
-   * @default - false, unless the '@aws-cdk/aws-kms:defaultKeyPolicies' feature flag is set.
+   * @default - false, unless the `@aws-cdk/aws-kms:defaultKeyPolicies` feature flag is set.
    * @see https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html#key-policy-default-allow-root-enable-iam
-   * @deprecated redundant with the '@aws-cdk/aws-kms:defaultKeyPolicies' feature flag
-   */
+   * @deprecated redundant with the `@aws-cdk/aws-kms:defaultKeyPolicies` feature flag
+   */
```
Comment (njlynch) [inline packages/@aws-cdk/aws-kms/lib/key.ts:331]:
```suggestion
   */
```

Comment (michaelbrewer) [reply packages/@aws-cdk/aws-kms/lib/key.ts]: @njlynch  I have patched it

**Review by njlynch — DISMISSED** [review]

**Comment by njlynch** [comment]
@michaelbrewer  - Can you allow changes to be pushed to your fork (see above mergify comment)? Without that, we can't update from the base branch and merge.

**Comment by michaelbrewer** [comment]
> @michaelbrewer - Can you allow changes to be pushed to your fork (see above mergify comment)? Without that, we can't update from the base branch and merge.

That option is not available to me, see screenshot:

Screen Shot 2021-02-22 at 4 39 47 AM

**Comment by michaelbrewer** [comment]
> @michaelbrewer - Can you allow changes to be pushed to your fork (see above mergify comment)? Without that, we can't update from the base branch and merge.

This was forked from a team account : https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847/11

**Comment by michaelbrewer** [comment]
@iliapolo  moving to : https://github.com/aws/aws-cdk/pull/13199

**Pull request closed without merging**